app/script/media/MediaStreamHandler.js

```diff
@@ -345,25 +345,31 @@ z.media.MediaStreamHandler = class MediaStreamHandler {
    *
    * @private
    * @param {z.media.MediaType} mediaType - Requested media type
-   * @returns {Promise} Resolves when the permissions are not denied
+   * @returns {Promise} Resolves true when permissions need to be prompted for
```

Comment:
The API of this method is confusing.
I would suggest to rename it to something like `hasPermissionToAccess(mediaType)` and reverse the returned boolean values